{"uuid": "fb5e7bc7-26b5-4463-a2a9-7f186dd58d32", "children": ["ea0cae8d-c8d4-406a-980e-878889f5ab6d"], "befores": [{"name": "setup", "status": "broken", "statusDetails": {"message": "selenium.common.exceptions.SessionNotCreatedException: Message: session not created: DevToolsActivePort file doesn't exist\nStacktrace:\n\tGetHandleVerifier [0x00BDE123+48179]\n\t(No symbol) [0x00B65D01]\n\t(No symbol) [0x00A4D72D]\n\t(No symbol) [0x00A7808E]\n\t(No symbol) [0x00A73C80]\n\t(No symbol) [0x00A70A69]\n\t(No symbol) [0x00AACA51]\n\t(No symbol) [0x00AAC5FA]\n\t(No symbol) [0x00AA3A26]\n\t(No symbol) [0x00A7B7BC]\n\t(No symbol) [0x00A7C62D]\n\tGetHandleVerifier [0x00EF7C33+3299139]\n\tGetHandleVerifier [0x00F35BF2+3553026]\n\tGetHandleVerifier [0x00F30BCC+3532508]\n\tGetHandleVerifier [0x00C7494E+664670]\n\t(No symbol) [0x00B70AB4]\n\t(No symbol) [0x00B6BF08]\n\t(No symbol) [0x00B6C02D]\n\t(No symbol) [0x00B5DD00]\n\tBaseThreadInitThunk [0x756B6839+25]\n\tRtlGetFullPathName_UEx [0x7769906F+1215]\n\tRtlGetFullPathName_UEx [0x7769903D+1165]\n\n", "trace": "  File \"D:\\Cred_Rev2\\TestFrameworkDemo\\venv\\Lib\\site-packages\\pluggy\\_callers.py\", line 77, in _multicall\n    res = hook_impl.function(*args)\n          ^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"D:\\Cred_Rev2\\TestFrameworkDemo\\venv\\Lib\\site-packages\\_pytest\\fixtures.py\", line 1123, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"D:\\Cred_Rev2\\TestFrameworkDemo\\venv\\Lib\\site-packages\\_pytest\\fixtures.py\", line 895, in call_fixture_func\n    fixture_result = next(generator)\n                     ^^^^^^^^^^^^^^^\n  File \"D:\\Cred_Rev2\\TestFrameworkDemo\\testcases\\conftest.py\", line 25, in setup\n    driver = webdriver.Chrome(service=ChromeService(ChromeDriverManager().install()), options=options)\n             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"D:\\Cred_Rev2\\TestFrameworkDemo\\venv\\Lib\\site-packages\\selenium\\webdriver\\chrome\\webdriver.py\", line 45, in __init__\n    super().__init__(\n  File \"D:\\Cred_Rev2\\TestFrameworkDemo\\venv\\Lib\\site-packages\\selenium\\webdriver\\chromium\\webdriver.py\", line 61, in __init__\n    super().__init__(command_executor=executor, options=options)\n  File \"D:\\Cred_Rev2\\TestFrameworkDemo\\venv\\Lib\\site-packages\\selenium\\webdriver\\remote\\webdriver.py\", line 209, in __init__\n    self.start_session(capabilities)\n  File \"D:\\Cred_Rev2\\TestFrameworkDemo\\venv\\Lib\\site-packages\\selenium\\webdriver\\remote\\webdriver.py\", line 293, in start_session\n    response = self.execute(Command.NEW_SESSION, caps)[\"value\"]\n               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"D:\\Cred_Rev2\\TestFrameworkDemo\\venv\\Lib\\site-packages\\selenium\\webdriver\\remote\\webdriver.py\", line 348, in execute\n    self.error_handler.check_response(response)\n  File \"D:\\Cred_Rev2\\TestFrameworkDemo\\venv\\Lib\\site-packages\\selenium\\webdriver\\remote\\errorhandler.py\", line 229, in check_response\n    raise exception_class(message, screen, stacktrace)\n"}, "start": 1708519139827, "stop": 1708519262355}], "start": 1708519139827, "stop": 1708519270326}